lib/LaTeXML/Engine/TeX_Math.pool.ltxml

```diff
@@ -732,20 +732,21 @@ our %DELIMITER_MAP =
   "\x{21D5}" => { lrole => 'OPEN',    rrole => 'CLOSE', name => 'Updownarrow' },    # ??
   );
 
+sub checkDelimiterHints {
+  my ($stomach, $whatsit) = @_;
+  $whatsit->setProperty(hint => 1) if ToString($whatsit->getArg(1)) eq '.';
+  return; }
+
```

Comment:
In fact, since we seem to be accumulating layers of helpers, it's almost like we're not getting any benefit from `TeXDelimiter`. Maybe `TeXDelimiter` should already recognize "." and convert it to a whatsit for `\lx@empty@delimiter`, which constructs an `ltx:XMHint`? Then all the constructors are simple `#`, and they don't need `checkDelimiterHints` or a hint property. We'd still have to tweak `addDelimiterRole` and `augmentDelimiterProperties` to recognize the hint. Whadya think?